### PR TITLE
Switch default font to Lato

### PR DESCRIPTION
**Pull request by mariosant**
When I first opened brunch.io, I thought 'Wow, that typography needs some fixing'! This PR is for fixing that. Please have in mind that I am no designer, so I just switched the default font to [Lato](https://www.google.com/fonts/specimen/Lato). I believe it looks much better that way.

screenshot

### CODE REVIEW

**Comment by paulmillr** [comment]
Not a big fan of Lato i'm afraid — but let's switch the default to `-apple-system`. And something else for other systems.

**Comment by mariosant** [comment]
Sorry @paulmillr, not sure if I understood correctly. Should we change to an apple used default font, then fallback to something else? Like, `Helvetica Lato sans-serif`?

**Comment by mariosant** [comment]
@paulmillr just figured it out. I had no idea that this is possible.

https://www.webkit.org/blog/3709/using-the-system-font-in-web-content/

**Comment by mariosant** [comment]
I guess it is ready to go, @paulmillr !

**Comment by paulmillr** [comment]
Thanks!

Actually i've meant using some kind of default font on all systems, besides from only apple. So, `apple-system` for macs and something else (Helvetica Neue?) for other stuff.

**Comment by mariosant** [comment]
Hehehe, ok. I guess it should be `Dejavu-sans` for linux systems, but what about windows systems? Arial should do the trick?

**Comment by mariosant** [comment]
Oh and there's `Calibri`!

File: app/styles/style.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+html, body {
+  font-family: -apple-system 'Dejavu Sans' 'Calibri' 'Arial 'sans-serif;
```
Comment (paulmillr) [inline app/styles/style.scss]: Use commas between font names

File: app/_layouts/_default.jade [hunk 2]
```diff
@@ -10,6 +10,7 @@ html
     meta(name='viewport', content='width=device-width, initial-scale=1')
     link(rel='shortcut icon', type="image/x-icon", href='/favicon.ico')
     link(rel='stylesheet', href='styles/app.css')
+    link(rel='stylesheet', href='styles/style.css')
```
Comment (paulmillr) [inline app/_layouts/_default.jade]: Can we put it inside one `css` file?

Comment (mariosant) [inline app/_layouts/_default.jade]: Not sure how to do that, will have to dig it a little more. I could not find the `app.scss` file. How is it generated?

Comment (paulmillr) [inline app/_layouts/_default.jade]: https://github.com/brunch/brunch.github.io/blob/brunch/app/assets/styles/app.css

**Comment by paulmillr** [comment]
done

**Pull request closed without merging**